plugins/systemd/systemd.plugin.zsh

```diff
@@ -20,3 +20,14 @@ alias sc-mask-now="sc-mask --now"
 alias scu-enable-now="scu-enable --now"
 alias scu-disable-now="scu-disable --now"
 alias scu-mask-now="scu-mask --now"
+
+function sc-prompt-status {
+  for UNIT in $@
+  do
+    [ -n "$ZSH_THEME_SYSTEMD_PROMPT_CAPS" ] && UNITDISPLAY=$(tr '[:lower:]' '[:upper:]' <<< $UNIT) || UNITDISPLAY=$UNIT
+    echo -n "$ZSH_THEME_SYSTEMD_PROMPT_PREFIX$UNITDISPLAY:"
+    [ "$(sc-is-active $UNIT)" = "active" ] && echo -n $ZSH_THEME_SYSTEMD_PROMPT_ACTIVE || echo -n $ZSH_THEME_SYSTEMD_PROMPT_NOTACTIVE
```

Comment:
Use the full command here. Also check for the exit code: `systemctl is-active $unit` returns 0 when the unit is active, other codes if not. That enables you to put this in an `if`, which is better looking:
```zsh
if systemctl is-active $unit; then
  echo -n $ZSH_THEME_SYSTEMD_PROMPT_ACTIVE
else
  echo -n $ZSH_THEME_SYSTEMD_PROMPT_NOTACTIVE
fi
```